feat: adding format file by props to upload input (#33)
vtex-apps/store-form: docs/README.md

### `form-input.upload` props

| Prop name  | Type | Description                                                                                                                                                                                                                                                                                                           | Default Value |
| ---------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
| `pointer`  | `string` | ![https://img.shields.io/badge/-Mandatory-red](https://img.shields.io/badge/-Mandatory-red) JSON schema pointer i.e. the JSON schema path  (for example: #/properties/address) in which the form block inputs should be validated against. Note that since you are configuring a `form-field-group` block, the path must not include a schema's sub-property, only a schema's property. | `undefined`     |

> | `typeAceppt` | `string` | ![https://img.shields.io/badge/-optional-yellow](https://img.shields.io/badge/-optional-yellow) By default the upload input just accept image and pdf format files. If you want to customize it, you can pass the format type that you want following this pattern: `*.TYPEFILE`. You can also pass multiple format types if you want: `*.TYPEFILE1, *.TYPEFILE2`

Reviewer: ```suggestion
| `accept` | `string` | ![https://img.shields.io/badge/-optional-yellow](https://img.shields.io/badge/-optional-yellow) By default the upload input just accept image and PDF format files. If you want to customize it, you can pass the format type that you want following this pattern: `*.TYPEFILE`. You can [read more about the `accept` field](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#accept).
```
